atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/qos/ThriftQueryWeighers.java

```diff
@@ -69,6 +71,13 @@ private ThriftQueryWeighers() { }
         return writeWeigher(numRows, () -> ThriftObjectSizeUtils.getApproximateWriteByteCount(mutationMap));
     }
 
+    public static QosClient.QueryWeigher<CASResult> cas(List<Column> updates) {
+        // TODO(nziebart): technically CAS involves both reads and writes; currently we are just counting it as a read
```

Comment:
It sounds trivial to count both the reads and writes, since we know the content that we expect to read. Let's just do it (TM).